### ISSUE TITLE
Bad behaviour when using typings and optional argument in function signature 

### ISSUE THREAD

**Issue by half1red** [google/yapf#463]
Hi, 

There is an issue with yapf when in signature one use optional argument and typings:

```
def foo(bar, nb: int = 1):
    pass
```

becomes:
```
def foo(bar, nb:int=1):
    pass
```

which is not what is specified in PEP526.

I have not found a way to set this right. This fires a pylint warning C0326.

(Sorry for the poor english)

**Comment by gwelymernans** [comment]
Which version of YAPF are you using? This was fixed in v0.18.0.

**Comment by half1red** [comment]
My bad, I am using v0.17, the one available on conda-forge.
